plugins.xml

```diff
@@ -1864,6 +1864,19 @@
 			</compatibility>
 			<certification type="partner"/><description>This release of the Shariff plugin for OJS/OMP 3.3.0 fixes a compatibility issue with PHP 8.</description>
 		</release>
+		<release date="2023-02-21" version="3.4.0.0" md5="319d33932cc130e592768e6d9575f63a">
+			<package>https://github.com/ojsde/shariff/releases/download/v3_4_0/shariff-3_4_0.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.4.0.0</version>
```

Comment:
@ronste, I'd suggest using: `~3.4.0.0` for the version declarations.
That way we won't have to update this for 3.4.0-1, 3.4.0-2, ...

Reply:
I updated the PR accordingly